src/building/bootstrapping.md

```diff
@@ -264,6 +264,55 @@ is built by the _beta_ compiler, but using the _master_ version of libstd!
 The only time `rustc` uses `cfg(bootstrap)` is when it adds internal lints
 that use diagnostic items. This happens very rarely.
 
+### What is a 'sysroot'?
+
+When you build a project with cargo, the build artifacts for dependendencies
+are normally stored in `target/debug/deps`. This only contains dependencies cargo
+knows about; in particular, it doesn't have the standard library. Where do
+`std` or `proc_macro` come from? It comes from the **sysroot**, a directory
+where the compiler loads build artifacts at runtime. The sysroot doesn't just
```

Comment:
The sysroot is also not really a directory but rather the "root" of a bunch of related directories? Notably for example rustc --print sysroot contains no loadable aritfacts, you need to go to rustlib/$target/lib for that...